src/react/databrowser/objects/details/ObjectRestorationButtonAndModal.tsx

```diff
@@ -97,8 +90,9 @@ const ObjectRestorationButtonAndModal = ({
                 setRestorationDays(parseInt(e.target.value))
               }
               size="1/3"
+              min={1}
             ></Input>{' '}
-            day(s)
+            day{restorationDays >= 2 ? '(s)' : ''}
```

Comment:
I guess parentheses can be dropped then 😄

Reply:
<img src="https://media4.giphy.com/media/JQfyu8zlpWphJobAJV/giphy-downsized-medium.gif"/>